sources/engine/Stride.Physics/Simulation.cs

```diff
@@ -687,7 +688,7 @@ internal void Simulate(float deltaTime)
 
             SimulationProfiler = Profiler.Begin(PhysicsProfilingKeys.SimulationProfilingKey);
 
-            if (discreteDynamicsWorld != null) discreteDynamicsWorld.StepSimulation(deltaTime, MaxSubSteps, FixedTimeStep);
+            if (discreteDynamicsWorld != null) discreteDynamicsWorld.StepSimulation(deltaTime, MaxSubSteps, FixedTimeStep * (float) GameTime.Factor);
```

Comment:
here you need to multiply the deltaTime with the TimeFactor instead of the FixedTimeStep as multiplying the FixedTimeStep will mess up the simulation

this video shows how the physics act weirdly when slowing time https://www.youtube.com/watch?v=xj5wuY6jrWw&feature=youtu.be (the ball is supposed to keep bouncing forever)